pkg/controller/add_checlusterrestore.go

```diff
@@ -0,0 +1,10 @@
+package controller
```

Comment:
some files miss license header

Reply:
They are autogenerated, so I didn't do it, but will add the license, makes sense.